boards/nxp/imx95_evk/CMakeLists.txt

```diff
@@ -11,3 +11,39 @@ if (CONFIG_SOF AND CONFIG_BOARD_IMX95_EVK_MIMX9596_M7_DDR)
     DEPENDS ${CMAKE_BINARY_DIR}/zephyr/${KERNEL_ELF_NAME}
   )
 endif()
+
+if(CONFIG_BOARD_NXP_SPSDK_IMAGE OR (DEFINED ENV{CONFIG_BOARD_NXP_SPSDK_IMAGE}
+  AND "$ENV{CONFIG_BOARD_NXP_SPSDK_IMAGE}" STREQUAL "y"))
+  if(CONFIG_BOARD_IMX95_EVK_MIMX9596_M7)
+    file(WRITE ${CMAKE_BINARY_DIR}/zephyr/imx95_evk_mimx9596_m7_ahab.yaml
+      "
+      family: mimx9596
+      revision: a1
+      target_memory: standard
+      output:                   ${CMAKE_BINARY_DIR}/zephyr/flash.bin
+      containers:
+        - binary_container:
+            path:               ${CMAKE_BINARY_DIR}/zephyr/imx95-19x19-lpddr5-evk-boot-firmware-0.1/mx95a0-ahab-container.img
+        - container:
+            srk_set: none
+            images:
+              - lpddr_imem:     ${CMAKE_BINARY_DIR}/zephyr/imx95-19x19-lpddr5-evk-boot-firmware-0.1/lpddr5_imem_v202311.bin
```

Comment:
for "imx95-19x19-lpddr5-evk-boot-firmware-0.1", suggest to define a variable for firmware image name or version name, so it will be easier to update in case of version or image name update.

Reply:
OKay, updated using variable.